### ISSUE TITLE
Trident What's New page shows: 'Removed support for EOL Windows Server 2019', but shows Windows 2019 in other doc links

### ISSUE THREAD

**Issue by crossond** [NetAppDocs/trident#455]
### Page URL

https://docs.netapp.com/us-en/trident/trident-rn.html

### Page title

What’s new

### Summary

Trident 'What's New page'  for 24.06 Release shows: 'Removed support for EOL Windows Server 2019', but shows Windows 2019 in other doc links
     - What does 'Removed Support' mean?  
     - All other doc pages still reference Windows 2019 Server as the only Windows OS that is supported.
     - I've listed many of these pages below, but a full scan of the Trident doc should be searched to correct this.
This is confusing.  Support and Customers need clarification on what this means..

https://docs.netapp.com/us-en/trident/trident-use/ontap-nas-prep.html?q=windows+2019#dynamically-manage-export-policies
https://docs.netapp.com/us-en/trident/trident-use/ontap-nas-prep.html?q=windows+2019#prepare-to-provision-smb-volumes
https://docs.netapp.com/us-en/trident/trident-get-started/requirements.html#tested-host-operating-systems
https://docs.netapp.com/us-en/trident/trident-use/anf-prep.html#prerequisites-for-nfs-and-smb-volumes
https://docs.netapp.com/us-en/trident/trident-use/trident-fsx-prep.html#prepare-to-provision-smb-volumes

### Public issues must not contain sensitive information

- [X] This issue contains no sensitive information.

**Comment by netapp-aruldeepa** [comment]
Hi @crossond, 

Thanks for bringing this up.
I will work with my team to review the documentation, and update as required.

Thanks,
Deepa

**Comment by crossond** [comment]
Any update on this"?

**Comment by netapp-aruldeepa** [comment]
Hi @crossond, 

The document has been updated and repushed.
Please let us know if we can close this issue.

Thanks,
Deepa

**Comment by crossond** [comment]
Thanks for updating the docs.    This issue has hot customer impact.  I've posted the below to the engineering trident slack channel for a response: 
-
I received notification that the Trident doc issue https://github.com/NetAppDocs/trident/issues/455#issuecomment-2236207868 is now updated.
And confirmed the Trident Docs now show 'Windows 2019 is Deprecated', and 'Windows 2022 is Supported'.
All of the appropriate pages regarding Windows Servers have also been updated, removing any mention of 'Windows 2019 Server' and adding 'Windows 2022' in place.
**This is very abrupt change for customers already using Windows 2019.**

- Does this mean upgrade to 24.06 will no longer pull binaries for Windows 2019 nodes? 

- Can we confirm what 'Deprecated in 24.06' actually means? 

Customers will be taken unaware, like Customer JackHenry and Assoc https://smartsolvecm.netapp.com/ui/#/case/2010098405.
They’ve hit an issue during upgrade to 24.06 where Windows Server 2022 pods are Running, but Windows 2019 pods are in CrashLoopBackOff trying to load the image.

Customer stated today:
"We’ll need an answer ASAP from your engineering team. Not being able to support Windows workloads within a GCP GKE environment with the Trident CSI driver is going to be a show-stopper!"
